apps/sms/views/conversation/js/conversation.js

```diff
@@ -634,8 +651,8 @@ var ConversationView = {
       });
     }
 
-    // Let's mark thread only when inbox is fully rendered and target node
-    // is in the DOM tree.
+    // Let's mark the conversation only when inbox is fully rendered and
+    // the target node is in the DOM tree.
```

Comment:
__unrelated note:__ nasty dependency that we'll be able to remove with ConversationService event :)